mermaid.config.json

```diff
@@ -1,9 +1,8 @@
   "deterministicIds": true,
-  "fontFamily": "Noto Sans",
-  "actorFontFamily": "Noto Sans",
-  "noteFontFamily": "Noto Sans",
-  "messageFontFamily": "Noto Sans",
-  "personFontFamily": "Noto Sans"
+  "themeVariables": {
+    "// background": "bug. ignored. we use --backgroundColor transparent in cli instead, and override in themeCSS"
```

Comment:
Is this reported upstream? Do we just have to bump mermaid CLI?

Reply:
It seems 70% of it is broken so I haven't reported upstream